### ISSUE TITLE
Meyda License

### ISSUE THREAD

**Issue by ajaska** [meyda/meyda#396]
Hi, I was looking at using this code as the basis for a similarly MIT-licensed project, but I noticed the documentation states that major portions of the project have been adapted from [yaafe](http://yaafe.sourceforge.net/) which is a LGPLv3 project. The GPL family of licenses (notoriously) does not permit GPL source code and derivatives thereof being included in an MIT-licensed project (according to the GPL license, the ENTIRE project must become licensed under GPL if ANY code is GPL).

I'm wondering if this may have been an oversight of the original authors (I don't expect everyone to be familiar with the nuances of software licensing) or if the original copyright holders of yaafe may have provided an exception here (in which case it would be helpful to have a notice of such)? Or maybe I'm mistaken about the extent to which GPL-licensed code has been used here?

Please don't take this the wrong way, it's not my intention to police licensing of open source projects on the internet, intimidate you, etc. I find myself frequently frustrated by the GPL but don't want to run afoul of its restrictions in the software I am producing. If the licensing situation here is murky, I can just move along and not be any more of a nuisance 😛.

**Comment by hughrawlinson** [comment]
Hi!

We probably didn't consider this when we started working on this. Meyda doesn't include any of Yaafe's code, and since the feature extractors in Meyda and Yaafe are implementations of algorithms that are defined externally to both libraries, I don't think there's a violation of Yaafe's licenses here. I think we looked at Yaafe's implementations to make sure that we didn't get any code wrong, but that's the extent of it.

Hope that helps! Let me know if you have any further questions and I'll reopen the ticket.
